akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -230,6 +233,8 @@ private[akka] object Running {
   // Needed for WithSeqNrAccessible, when unstashing
   private var _currentSequenceNumber = 0L
 
+  private var recursiveUnstashOne = 0
```

Comment:
I considered adding this in the StashManagement trait instead but that becomes difficult with the switching of behaviors (HandlingCommands -> WaitingForContinueUnstash -> HandlingCommands)